### PR TITLE
Impose list limit on all entity types.

### PR DESCRIPTION
**Pull request by rabbah**
Allow asset listing in namespace to skip and limit.

### CODE REVIEW

**Comment by rabbah** [comment]
pg3/1634 🏃

File: core/controller/src/main/scala/whisk/core/entitlement/Collection.scala [hunk 1]
```diff
@@ -109,6 +110,10 @@ protected[core] object Collection {
 
   protected[core] def requiredProperties = WhiskEntityStore.requiredProperties
 
+  /** Number of records allowed per query. */
+  protected[core] val DEFAULT_LIST_LIMIT = 30
```
Comment (dubeejw) [inline core/controller/src/main/scala/whisk/core/entitlement/Collection.scala]: Would be nice if `DEFAULT_LIST_LIMIT` and `MAX_LIST_LIMIT` were configurable with pureconfig, but we can do that later.

**Comment by dubeejw** [comment]
Travis timed-out, restarted.

**Comment by dubeejw** [comment]
Test failures:

```
whisk.core.cli.test.WskBasicUsageTests > Wsk invalid argument handling should reject commands that are executed with invalid arguments FAILED
    org.scalatest.exceptions.TestFailedException: List(namespace, get, namespace, invalidArg) "error: Invalid argument(s): namespace, invalidArg. No arguments are required.
    Run 'wsk --help' for usage.
    " did not include substring "error: Invalid argument(s): invalidArg. An optional namespace is the only valid argument."
```

```
system.basic.WskBasicTests > Wsk Namespace CLI should not list entities with an invalid namespace FAILED
    org.scalatest.exceptions.TestFailedException: /home/travis/build/apache/incubator-openwhisk/ansible/../bin/wsk -i --apihost 172.17.0.1 --apiversion v1 namespace get /fakeNamespace --auth 23bc46b1-71f6-4ed5-8c54-816aa4f8c502:123zO3xZCLrMN6v2BKK1dXYFpXlPkccOFqm12CdAsMgRU4VrNZ9lyGVCGuMDGIwP
    error: Invalid argument(s): /fakeNamespace. No arguments are required.
    Run 'wsk --help' for usage.
    exit code: 1 was not equal to 147
```

**Pull request merged**